### DISCUSSION TITLE
Welcome to DelHel Discussions!

### DISCUSSION THREAD

**Discussion by MorpheusXAUT** [MorpheusXAUT/DelHel#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  Feel free to put any questions you have regarding the development of DelHel or any of its config files here before opening an issue.

**Comment by rechbergerd** [comment]
Hi!
Any chance, any of the following ideas could be considered?
- Spreading of route-config-files in a "routes.d" sub-directory of pwd (yeah, I am a linux guy too -.-)
- Spreading of airport-config-files in a "airport.d" sub-directory of pwd
- Making a centralized, versioned (AIRAC+subversion, similar to aero-nav) repository of the configs -- for infrastructure (server (HTTPS)) hit me up
- Easier distribution of config-files through the repository?

Kindest

**Reply by MorpheusXAUT** [reply]
Hi, sorry for the late reply!

1) and 2) are definitely options, however I'm not sure they'd warrant the work required to support them, as the complexity of the route and airport config files so far are still remains relatively low (I'd hope). Since the config is most likely provided by/built for a VACC, I'd hope it can be somewhat contained within a single file without too much trouble.
I'm happy to keep this in mind and am glad to accept a PR implementing this, but currently unfortunately don't have time to implement it myself.

3) and 4) are two good points I have considered as well - allowing for the config files to be distributed independently and updated by the plugin itself without any sector file updates or similar. Once again, I'm not sure the extra work required to implement this is justified given the frequency at which SIDs and routes change plus the fact that most people will probably use DelHel within one VACC instead of managing multiple configs for different regions in one setup.

I'm always open for suggestions and feedback, but my time available to DelHel is quite limited at the moment unfortunately, so not everything will "make the cut". I'm more than happy to take a look at PRs though, should you or someone else find time to implement it 😄
